tasks/level-1/1.4.2.yml

```diff
@@ -33,3 +33,14 @@
     - "1.4.2"
     - scored
 
+- name: 1.4.2 - Ensure rescue.service and emergency.service both use /sbin/sulogin for ExecStart
+  command: grep -xq 'ExecStart=-/bin/sh -c "/usr/sbin/sulogin; /usr/bin/systemctl --fail --no-block default"' /usr/lib/systemd/system/{{ item }}.service
```

Comment:
So far as I can tell this will only fail the ansible play if grep returns a non-zero exit code.

Instead the files should be updated to ensure that they **do** use /sbin/sulogin for ExecStart